common/djangoapps/student/helpers.py

```diff
@@ -368,6 +368,21 @@ def get_redirect_to(request):
     return redirect_to
 
 
+def _is_safe_redirect(request, redirect_to):
+    """
+    Determine if the given redirect URL/path is safe for redirection.
+    """
+    request_host = request.get_host()
```

Comment:
Add comment and possibly an additional test for courses.edx.org in the logout PR.